system-design/app-interoperability/observability.md

```diff
@@ -284,13 +287,13 @@ The following table shows the metrics emitted by the indicated receivers when us
 
 **Logs**
 
-The following shows the logs emitted by the indicated receiver. The Kubernetes Event receiver collects the event logs when using the default configuration. The Kubernetes Object Receiver and Log Receiver must be configured to collect the desired logs.
+The following shows the logs emitted by the indicated receiver. The Kubernetes Events receiver collects the event logs when using the default configuration. The Kubernetes Object Receiver and Log Receiver must be configured to collect the desired logs.
 
-| Source | [Kubernetes Object Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sobjectsreceiver) | [Kubernetes Event Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8seventsreceiver) | [File Log Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver) |
+| Source | [Kubernetes Object Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sobjectsreceiver) | [Kubernetes Events Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8seventsreceiver) | [File Log Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver) |
 |---------|--------------------------------|------------------------------|--------------------|
 | Container Logs |  |  | - |
-| K8s Events | - | - |  |  
-| K8s Resources | X |  |  |
+| K8s Events | - | X |  |  
+| K8s Resources | - |  |  |
```

Comment:
Can this be generalized? I'm slightly concerned consumers of the standard would take this as we picked a kubernetes distro. 

Recommend change from k8s to agnostic "kubernetes"

Reply:
updated